Specs/Scene/ExpressionSpec.js

```diff
@@ -814,6 +814,24 @@ defineSuite([
         expect(expression.evaluate(undefined)).toEqual(false);
     });
 
+    it('evaluates abs function', function() {
```

Comment:
> If possible, it would be nice to check if the shader compiles when using the GLSL backend. If dozens of tests like this slow down the unit tests too much, we could also change the test granularity to be more coarse, e.g., all trig functions, etc.

Yeah this would be better if it can be done simply. I'll save this for later though.